### PR TITLE
chore(LT-5853): [Lykke.Job.CandlesHistoryWriter] Update MessagePack t…

### PR DESCRIPTION
**Pull request by lykke-vashetsin**
…o 2.x version

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -9,24 +9,24 @@
...
     <PackageVersion Include="LykkeBiz.Logs.MsSql" Version="4.0.0" />
     <PackageVersion Include="LykkeBiz.Logs.Serilog" Version="3.3.3" />
-    <PackageVersion Include="LykkeBiz.RabbitMqBroker" Version="15.1.1" />
-    <PackageVersion Include="LykkeBiz.SettingsReader" Version="9.0.0" />
-    <PackageVersion Include="LykkeBiz.Snow.Cqrs" Version="5.1.0" />
-    <PackageVersion Include="MessagePack" Version="1.9.11" />
+    <PackageVersion Include="LykkeBiz.RabbitMqBroker" Version="16.0.0" />
+    <PackageVersion Include="LykkeBiz.SettingsReader" Version="9.0.1" />
+    <PackageVersion Include="LykkeBiz.Snow.Cqrs" Version="7.0.1" />
+    <PackageVersion Include="MessagePack" Version="2.5.187" />
```
Comment (tarurar) [inline Directory.Packages.props:26]: You probably do not need direct dependency on MessagePack.

File: Directory.Packages.props [hunk 2]
```diff
@@ -9,24 +9,24 @@
...
     <PackageVersion Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.22.0" />
     <PackageVersion Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="8.0.8" />
-    <PackageVersion Include="Microsoft.Extensions.Logging.Debug" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Logging.Debug" Version="8.0.1" />
```
Comment (tarurar) [inline Directory.Packages.props:29]: Why do we need debug package?

File: src/Lykke.Job.CandleHistoryWriter.Repositories/Candles/SqlCandlesHistoryRepository.cs [hunk 3]
```diff
@@ -110,12 +110,12 @@ public async Task ApplyRFactor(string productId, decimal rFactor, DateTime rFact
                 new UpdateOldWeeklyCandlesCommand(rFactor, rFactorDate),
             };
 
-            if (!rFactorDate.SameWeek(lastTradingDay, DayOfWeek.Monday))
```
Comment (tarurar) [inline src/Lykke.Job.CandleHistoryWriter.Repositories/Candles/SqlCandlesHistoryRepository.cs:113]: How does rfactor task related to messagepack update?

**Review by tarurar — CHANGES_REQUESTED** [review]

File: src/Lykke.Job.CandleHistoryWriter.Repositories/Candles/SqlCandlesHistoryRepository.cs (hunk 3 above)
Comment (lykke-vashetsin) [reply src/Lykke.Job.CandleHistoryWriter.Repositories/Candles/SqlCandlesHistoryRepository.cs:113]: there was conflict namespaces

File: Directory.Packages.props (hunk 2 above)
Comment (lykke-vashetsin) [reply Directory.Packages.props]: removed

File: src/Lykke.Job.CandleHistoryWriter.Repositories/Candles/SqlCandlesHistoryRepository.cs [hunk 4]
```diff
@@ -110,12 +110,12 @@ public async Task ApplyRFactor(string productId, decimal rFactor, DateTime rFact
...
+            if (!CandlesHistoryWriter.Core.Domain.Candles.DateTimeExtensions.SameWeek(rFactorDate, lastTradingDay, DayOfWeek.Monday))
```
Comment (tarurar) [inline src/Lykke.Job.CandleHistoryWriter.Repositories/Candles/SqlCandlesHistoryRepository.cs:113]: It looks like an extension method. Why can’t you just use it like `rFactorDate.SameWeel`?

**Review by tarurar — APPROVED** [review]

**Pull request merged**